cmd/pull.go

```diff
@@ -166,7 +176,7 @@ func pullMigration(client *hasura.Client, name string) (hasura.Migration, error)
 
 		log.Debug("Creating initial migration")
 
-		migration.Data, err = client.Migration(migrationTables)
+		migration.Data, err = client.Migration(pgDumpSchemasFlags(schemas))
```

Comment:
Just looking at the variable names, did we send a list of tables before, and are sending a list of schemas now?

Will that cause problems?

Reply:
it wasn't a list of tables but a list of flags, f.e.

```
[]string{"--table", "my_table", "--table", "other_table"}
```

Reply:
Thanks for the clarification 👍